doc/en/changelog.rst

```diff
@@ -4842,7 +4870,7 @@ time or change existing behaviors in order to make them less surprising/more use
 * Updated docstrings with a more uniform style.
 
 * Add stderr write for ``pytest.exit(msg)`` during startup. Previously the message was never shown.
-  Thanks `@BeyondEvil`_ for reporting `#1210`_. Thanks to `@jgsonesen`_ and
+  Thanks `@BeyondEvil`_ for reporting `#1210`_. Thanks to @jgsonesen and
```

Comment:
@jgsonesen is a broken link

Reply:
For reference: was even changed before apparently already.  Original PR: https://github.com/pytest-dev/pytest/pull/1763.

Reply:
👍